backtracking/Magnet_Puzzle.py3

```diff
@@ -0,0 +1,194 @@
+# Write Python3 code here
```

Comment:
Plagiarized from [GeeksforGeeks](https://www.geeksforgeeks.org/magnet-puzzle-backtracking-9/). Code is identical down to the comments, only differing in some whitespace.

@cclauss Thoughts?